ooasp/encodings/ooasp_simple.lp

```diff
@@ -21,10 +21,10 @@ check_permanent_cv. % Checks for permanent constraints apply
 #include "ooasp_symmetry.lp". % Symmetry breaking constraints
 #include "ooasp_smart_generation.lp". % Functions for smart generation
 
-% Show statements. Notice that anything that is used in the smart generation functions
+% !!!!!!!!!!!! Show statements. Notice that anything that is used in the smart generation functions
 % as well as clinguin UI encoding should be shown to avoid incorrect results.
-#show object_needed/6.
-#show global_ub/3.
-#show global_lb/3.
-#show association_needed/4.
-#show .
\ No newline at end of file
+% #show object_needed/6.
+% #show global_ub/3.
+% #show global_lb/3.
+% #show association_needed/4.
+% #show .
```

Comment:
Why is this commented-out?

Reply:
So that everything is shown in this version, but we have a note on what are the needed things. @susuhahnml please correct me if I am wrong.

Reply:
This was addressed in Teams right? I see the last version working well.